File: C/diffSqr.c
/*
  Find the difference between the sum of the squares of the first one hundred natural numbers and the square of the sum.
*/

#include <stdio.h>
#define MAXVAL 100
int main(void)
{
    unsigned int summSqr = 0;  //Sum of squares
    unsigned int sqrSumm = 0;  //Square of the sum

    for (int i = 1; i <= MAXVAL; i++){
        summSqr += i * i;
        sqrSumm += i;
    }

    sqrSumm = sqrSumm * sqrSumm;
    printf("Difference between %d and %d is %d \n", 
            sqrSumm, summSqr, sqrSumm - summSqr);
    return 0;
}

File: Python/diffSqr.py
#  Find the difference between the sum of the squares of the first one hundred natural numbers and the square of the sum.

def main():
    summSqr = 0
    sqrSumm = 0

    for i in range(1, 101):
        summSqr += i * i
        sqrSumm += i

    sqrSumm = sqrSumm * sqrSumm
    print("Difference between {} and {} is {}".format(sqrSumm, \
           summSqr, sqrSumm - summSqr))

if __name__ == "__main__":
    main()
